src/pages/ProjectSettings.tsx

```diff
@@ -197,3 +203,7 @@ const tabRouteValues = Object.values(ProjectSettingsTabRoutes);
 const PageContainer = styled.div`
   display: flex;
 `;
+
+const DropdownContainer = styled.div`
+  padding: 0.5rem;
```

Comment:
can you use one of the [tokens instead](https://github.com/evergreen-ci/spruce/blob/main/src/constants/tokens.ts). `size.xs`  is the equivalent to 0.5rem